images/installer/Dockerfile

```diff
@@ -40,12 +71,13 @@ ENV PLAYBOOK_FILE=playbooks/byo/openshift_facts.yml \
 # /tmp/src (as per the source-to-image specs) so we import it there
 ADD . /tmp/src
 
-# Running the 'assemble' script provided by playbook2image will install
+# Running the 'assemble' script will install
 # dependencies specified in requirements.txt and install the 'oc' client
 # as per the INSTALL_OC environment setting above
-RUN /usr/libexec/s2i/assemble
+RUN /usr/bin/assemble
```

Comment:
we don't need any of this assemble stuff

I'm wondering what we need the `oc` client for either but I guess we should include that in case something is actually using it.

we don't need the `INSTALL_OC` variable if we just hardcode that bit from `assemble`